geometry_msgs/Pose2D center      # x, y, theta[m]
geometry_msgs/Vector3 velocity  # x,y,z [m/s]
float32[] data # gaussian 